operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -115,6 +115,13 @@ mimir:
           insecure: true
           secret_access_key: {{ .Values.minio.rootPassword }}
       {{- end }}
+      {{- if and (index .Values "admin-cache" "enabled") (eq (index .Values "admin-cache" "cacheType") "memcached" ) }}
+        cache_type: memcached
```

Comment:
I see you changed this to `backend_cache`. What do you think about only `backend`? that's how the GEM cache config is structured now (e.g. `-blocks-storage.bucket-store.index-cache.backend`)

Reply:
I needed to refactor backend-enterprise implementation to make allow using backend for the key config. Otherwise the keyname conflicted backend key for bucket storage. Please check the PR too if you are available. https://github.com/grafana/backend-enterprise/pull/4084